packages/@okta/vuepress-site/docs/guides/registration-inline-hook/resources/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Resources
+---
+
+For full descriptions of each object included in the request and response of the Registration Inline Hook requests, see the sections [Objects in the Request from Okta](/docs/reference/registration-hook/#objects-in-the-request-from-okta) and [Objects in the Response You Send](/docs/reference/registration-hook/#objects-in-response-you-send) in the reference documentation for the Registration Inline Hook.
+
+See the following other resources for information on Inline Hooks, Registration Inline Hooks, and Self-Service Registration:
+
+* See [Okta Hooks](https://www.okta.com/hooks/) for a high-level overivew of inline hooks
+* See [Inline Hooks](/docs/concepts/inline-hooks/) for a conceptual overview of how Okta Inline Hooks work.
+* See [Registration Inline Hook Reference](/docs/reference/registration-hook/) for a complete description of this Inline Hook type.
+* See [About self-service registration](https://help.okta.com/en/prod/Content/Topics/users-groups-profiles/usgp-self-service.htm) for further details on self-service.
```

Comment:
"further details on self-service registration."

Reply:
Weren't these links already covered at the beginning?

Reply:
Yes, two of them. I'm open to positioning this reference information in one location, either at the start or end.